playground/samples/propertyDependencies.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+  schema: {
+    title: "Property dependencies",
+    description: "These samples are best viewed without live validation.",
+    type: "object",
+    properties: {
+      unidirectional: {
+        title: "Unidirectional",
+        src:
+          "https://spacetelescope.github.io/understanding-json-schema/reference/object.html#dependencies",
+        description:
+          "In the following example, whenever a `credit_card` property is provided, a `billing_address` property must also be present.",
```

Comment:
Markdown is not supported, so backticks won't turn into `<code>` blocks, though you could use JSX instead of a string here.